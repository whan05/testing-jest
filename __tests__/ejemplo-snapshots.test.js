const cliente = {
    nombre: "Whanderley",
    balance: 500,
    tipo: "Premium"
}

describe('testing al cliente', () => { 
    test("El nombre es Whanderley", () => {
        expect(cliente).toMatchSnapshot();
    })
 })