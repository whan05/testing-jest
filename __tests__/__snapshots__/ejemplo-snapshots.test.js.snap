// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing al cliente El nombre es Whanderley 1`] = `
{
  "balance": 500,
  "nombre": "Whanderley",
  "tipo": "Premium",
}
`;
